ur10_moveit_config/CMakeLists.txt

```diff
@@ -5,5 +5,11 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
+if (CATKIN_ENABLE_TESTING)
+  find_package(roslaunch REQUIRED)
+  roslaunch_add_file_check(tests/moveit_planning_execution.xml)
+endif()
+
 install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
 install(DIRECTORY config DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY tests DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
Tests are typically not installed.

We don't install tests in any of our packages, so let's not start with these.

Reply:
Same for all other packages.